modules/Total/index.js

```diff
@@ -0,0 +1,28 @@
+const getEmote = (emote, discord) => {
+  return discord ? `:${emote}:` : emote;
+};
+
+const getBold = (string, discord) => {
+  return discord ? `**${string}**` : string;
+};
+
+module.exports = {
+  name: 'Total',
+  module(jaffamod) {
+    jaffamod.registerCommand('total', (message, reply, discord) => {
+      // Determine if it's JingleJam time
+      //  (December + first 7 days of the new year to view the total raised)
+      const d = new Date();
+      if (d.getMonth() === 11 || (d.getMonth() === 0 && d.getDate() <= 7)) {
```

Comment:
Is it fine to let this be available for the first 7 days of January?

Imo, it's a good reference as JingleJam might get mentioned in the first few streams in the new year etc.